tests/robot-tests/tests/public_api/public_api_minor_manual_changes.robot

```diff
@@ -163,7 +163,7 @@ Validate the version task statuses inside the 'Draft version task' section
 User clicks on Map locations link
     user clicks link    Map locations
     user waits until h3 is visible    Locations not found in new data set
-    user waits until element contains    xpath://table[@data-testid='mappable-table-region']/caption//strong[1]
+    user waits until element contains    css:[data-testid="mappable-table-region"] caption
```

Comment:
Was looking to find Robot changes :)  Nice one - looks like a good simplification too